packages/canvas-tokens/tokens/web/sys.json

```diff
@@ -816,6 +816,7 @@
         }
       }
     },
+    "layer-opacity": {},
```

Comment:
Is this part of the action token work?

Reply:
no, it's figma specific and should not been listed. I'm pushing PR to fix it